jupyterlab/labapp.py

```diff
@@ -573,6 +577,14 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
         help="Whether to expose the global app instance to browser via window.jupyterapp",
     )
 
+    custom_css = Bool(
+        True,
```

Comment:
Should the default for 4.1 be `True` or `False`? I would not ask this question if it was aimed at 5.0. I have no strong opinion, but believe it is important to discuss.

Reply:
Maybe `False` in JupyterLab to be on the conservative side? Users can opt-in easily since there is documentation that explains how to do so.